src/main/java/com/mercadopago/client/preference/PreferenceItemRequest.java

```diff
@@ -23,15 +26,19 @@ public class PreferenceItemRequest {
   /** Category of the item. */
   private final String categoryId;
 
+  /** ID da moeda. Código ISO_4217. */
```

Comment:
[req] Traduzir para inglês.
```suggestion
  /** Currency ID. Code ISO_4217. */
```